src/pages/Components/RatingStars/previews/stars-input-all-sizes.preview.scss

```diff
@@ -0,0 +1,11 @@
+@import 'settings-tools/_all-settings';
+@import 'components/_c.stars-input';
+
+.example {
+  display: block;
+  padding: 0 $mu150;
+
+  & > div {

```

Comment:
BEM please

Reply:
no type selectors, our example should follow our guidelines to

Reply:
Fixed